docs/user_guide/HOWTO/Using_the_SIMP_Release_RPM.rst

```diff
@@ -0,0 +1,126 @@
+.. _howto-use-the-simp-release-rpm:
+
+===========================
+SIMP Release RPM
+===========================
+
+Download Details
+---------------------------------
+
+Location
+********
+
+The SIMP Community Release RPM can be downloaded from our `server
+<https://download.simp-project.com/simp-release-community.rpm>`_.
+
+RPM Contents and Actions
+************************
+
+The RPM will do the following, which are detailed in sections below:
+
+- Create two Yum variable files to manage the SIMP repositories
+
+- Add the SIMP, EPEL, PostgreSQL, and Puppet repository files to the
```

Comment:
Might want to change these to something like "add the necessary repositories and GPG keys" so that we don't have to update it as things change over time.